tests/TestCase.php

```diff
@@ -30,7 +32,15 @@ protected function getPackageProviders($app)
 
     protected function defineDatabaseMigrations()
     {
-        $this->loadMigrationsFrom(__DIR__.'/../database/migrations');
+        $options = [
```

Comment:
`defineDatabaseMigrations` seems to be called for each test whereas resolving `Migrator` class seems to only happen for the first test in a test run. I was getting tests to pass when running individually but failing when running the full test suite.